NextGrowingTextView/NextGrowingTextView.swift

```diff
@@ -165,6 +165,7 @@ open class NextGrowingTextView: UIScrollView {
 
   private func setup() {
 
+    NotificationCenter.default.addObserver(self, selector: #selector(willShowMenu(_:)), name: UIMenuController.willShowMenuNotification, object: nil)
```

Comment:
Should not we need to set an object as _textView?
I thought it's better to set limitations of scope handling notifications that come from text views.

Reply:
ah, object would be UIMenuController, so my suggestion might be wrong

Reply:
Yes, we can't use the object parameter here :(

Reply:
I understood, thanks!